src/jshint.js

```diff
@@ -5071,12 +4993,16 @@ var JSHINT = (function() {
       },
 
       current: {
+        getlabel: function(t) {
```

Comment:
Looks like I don't know the codebase as well as I thought. Thanks for reminding me